index.html

```diff
@@ -38,19 +41,226 @@ <h1 class="boldtext">
+            <p>
+              A daily selection of privately<br />
+              personalized reads; no accounts or<br />
+              sign-ups required.
+            </p>
+          </div>
+
+          <div >
+            <ul class="languages">
+              <li class="word-style">html</li>
+              <li class="word-style">css</li>
+              <li class="word-style">javaScript</li>
+            </ul>
+          </div>
+        </div>
+        <div class="action">
+          <button type="button" class="action-button">See Project</button>
+        </div>
+      </section>
+
+      <!-- third card strts here -->
+      <section class="card-container card-type1">
+        <div class="image-container img3"></div>
+        <div class="content-container">
+          <h1 class="project-title">Tonic</h1>
+          <div class="frame2">
+            <h2>CANOPY</h2>
+            <ul class="card-role">
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+
+          <div class="description">
+            <p>
+              A daily selection of privately<br />
+              personalized reads; no accounts or<br />
+              sign-ups required.
+            </p>
+          </div>
+
+          <div >
+            <ul class="languages">
+              <li class="word-style">html</li>
+              <li class="word-style">css</li>
+              <li class="word-style">javaScript</li>
+            </ul>
+          </div>
+        </div>
+        <div class="action">
+          <button type="button" class="action-button">See Project</button>
+        </div>
+      </section>
+      
+
+      <!-- fourth card starts here -->
+      <section class="card-container card-type2">
+        <div class="image-container img4"></div>
+        <div class="content-container">
+          <h1 class="project-title">Multi-Post<br>
+            Stories</h1>
+          <div class="frame2">
+            <h2>CANOPY</h2>
+            <ul class="card-role">
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+
+          <div class="description">
+            <p>
+              A daily selection of privately<br />
+              personalized reads; no accounts or<br />
+              sign-ups required.
+            </p>
+          </div>
+
+          <div >
+            <ul class="languages">
+              <li class="word-style">html</li>
+              <li class="word-style">css</li>
+              <li class="word-style">javaScript</li>
+            </ul>
+          </div>
+        </div>
+        <div class="action">
+          <button type="button" class="action-button">See Project</button>
+        </div>
+      </section>
+    </section>
+
+      <!-- about my self section starts here -->
+      <section class="about-container">
+        <div class="about-myself">
+          <h1 class="about-heading">About<br>
+            Myself</h1>
```

Comment:
- [ ] Same comment as before, please change this `h1` tag to another `hx` tag. 😁